### PR TITLE
feat: [PIE-5901] Removed SPG_NG_GITHUB_WEBHOOK_AUTHENTICATION FF

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vcalasansh — APPROVED** [review]

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
retrigger cypress-pipeline

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Review by satyam-harness — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]

File: src/modules/72-triggers/pages/triggers/__tests__/WebhookTriggerConfigPanel.test.tsx [hunk 1]
```diff
@@ -62,26 +61,33 @@ function WrapperComponent(props: { initialValues: any; isEdit?: boolean }): JSX.
 
 describe('WebhookTriggerConfigPanel Triggers tests', () => {
   describe('Renders/snapshots', () => {
-    test('Initial Render - Github Trigger Configuration Panel with SPG_NG_GITHUB_WEBHOOK_AUTHENTICATION FF false', async () => {
+    test('Initial Render - Github Trigger Configuration Panel with Github Webhook Authentication Enabled', async () => {
```
Comment (AshwinBhatkal-Harness) [inline src/modules/72-triggers/pages/triggers/__tests__/WebhookTriggerConfigPanel.test.tsx:64]: How are checking enabled here? getTriggerConfigInitialValues?

**Pull request merged**